cfgov/legacy/views/complaint.py

```diff
@@ -48,15 +52,17 @@ def get_context_data(self, **kwargs):
     def get_ccdb_status_json(self, complaint_source):
         """Retrieve JSON describing the CCDB's status from a given URL."""
         try:
-            headers = {"accept": "application/json"}
-            res_json = requests.get(complaint_source, headers=headers).json()
-
-        except requests.exceptions.RequestException:
-            logger.exception("CCDB status data fetch failed.")
-            res_json = {}
+            args = {'field': 'all', 'size': '1', 'no_aggs': 'true'}
+            factory = APIRequestFactory()
+            request = factory.get('/search/', args, format='json')
```

Comment:
Is this the right path?  If it works, then I am impressed!

Reply:
Yeah, [here](https://github.com/cfpb/cfgov-refresh/blob/01bc0d677b475fdc7781917d6618a18f6127bf26/cfgov/cfgov/urls.py#L251) is where cfgov-refresh pulls in the ccdb5-api views